config.json

```diff
@@ -158,5 +158,19 @@
             "bucket": "opentoken-io-test-s3",
             "region": "us-east-1"
         }
+    },
+    "token": {
+        "idHash": {
+            "algorithm": "sha512",
+            "derivedLength": 32,
+            "encoding": "base64-uri",
+            "iterations": 4000,
```

Comment:
Is 4000 enough, I thought we'd want to do 10000 at least. But I see it's using sha512 so maybe 4000 is enough.